tidy3d/plugins/mode/mode_solver.py

```diff
@@ -1141,6 +1143,324 @@ def plot_field(
             **sel_kwargs,
         )
 
+    def plot(
+        self,
+        ax: Ax = None,
+        **patch_kwargs,
+    ) -> Ax:
+        """Plot each mode plane simulation's components.
+
+        Parameters
+        ----------
+        ax : matplotlib.axes._subplots.Axes = None
+            Matplotlib axes to plot on, if not specified, one is created.
+
+        Returns
+        -------
+        matplotlib.axes._subplots.Axes
+            The supplied or created matplotlib axes.
+
+        See Also
+        ---------
+
+        **Notebooks**
+            * `Visualizing geometries in Tidy3D: Plotting Materials <../../notebooks/VizSimulation.html#Plotting-Materials>`_
+
+        """
+        # Get the mode plane normal axis, center, and limits.
+        axes = [0, 1, 2]
+        a_center = [None, None, None]
+        n_axis = self.normal_axis
+        axes.pop(n_axis)
+        a_center[n_axis] = self.plane.center[n_axis]
+        h_lim = [
+            a_center[n_axis] - self.plane.size[axes[0]] / 2,
+            a_center[n_axis] + self.plane.size[axes[0]] / 2,
+        ]
+        v_lim = [
+            a_center[n_axis] - self.plane.size[axes[1]] / 2,
+            a_center[n_axis] + self.plane.size[axes[1]] / 2,
+        ]
+
+        return self.simulation.plot(
+            x=a_center[0],
+            y=a_center[1],
+            z=a_center[2],
+            hlim=h_lim,
+            vlim=v_lim,
+            source_alpha=0,
+            monitor_alpha=0,
+            lumped_element_alpha=0,
+            ax=ax,
+            **patch_kwargs,
+        )
+
+    def plot_eps(
+        self,
+        freq: float = None,
+        ax: Ax = None,
+    ) -> Ax:
```

Comment:
Let's add an `alpha` parameter like in `Simulation.plot_eps`, it will be handy if we want to plot the fields and overlay the epsilon. https://github.com/flexcompute/tidy3d/blob/af371ad9e59d3038c8a0166f9d74abfca749d17f/tidy3d/components/simulation.py#L391

Reply:
Done!